Create Device_Power.md

garmstron42 opened the pull request.

garmstron42 committed:
Create Device_Power.md

garmstron42 committed:
Create ReadMe.md

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
first draft nearly done Device_Power.md

garmstron42 committed:
adding images Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Add files via upload

garmstron42 committed:
goodfaithDevice_Power.md

Kevinwulrich committed:
Fixed spelling mistakes

Why is everyone spelling figure as firgure

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Kevin&Ross_are_meanDevice_Power.md

Kevin and Ross made fun of me for naming things wrong

JRNeal2020 committed:
Update Device_Power.md

Fixed the "v"'s :)

garmstron42 committed:
Update Device_Power.md

fixed citations

JesseTNRoberts commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,189 @@
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The power system shall be connected to the emergency stop           |Conceptual Design |
+| 2   | The power system shall power the main/processor unit                |Conceptual Design |
```
What does that mean?

garmstron42 replied:
changed these to reflect the new system being included and made these more precise

JesseTNRoberts reviewed (commented):
Make sure the constraints are precise (put units and ranges on all of them).

FinlayPatoto committed:
Update Device_Power.md

FinlayPatoto committed:
WRONG THING

garmstron42 committed:
added_emDevice_Power.md

Added other system, made constraints more precise

garmstron42 committed:
Update Device_Power.md

JesseTNRoberts commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,190 @@
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The power system shall be controlled by an emergency stop which will de-energize the device power system that will in turn de-energize the interceptor itself. |Conceptual Design |
```
Even the controller? Is that a good idea?

garmstron42 replied:
This was a Devcom constraint. They specifically requested the emergency stop deenergize the entire interceptor. The emergency stop is really only there if the system takes a wrong turn and needs to be turned for safety reasons.

JesseTNRoberts replied:
Sure. But that doesn't change the question.

JesseTNRoberts commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,190 @@
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The power system shall be controlled by an emergency stop which will de-energize the device power system that will in turn de-energize the interceptor itself. |Conceptual Design |
+| 2   | The power system shall supply 15 Watts to the main/processor unit   |Conceptual Design |
```
Why?

JesseTNRoberts commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,190 @@
+
+This system will be the primary power supplier for the main/processor unit. It must be able to supply 15 Watts to this system as required by the main/processor.  
+
+<sup>3</sup> The power system shall supply 1.65 Watts to the communication receiver [Conceptual Design]
```
this is circular and without reason.

JesseTNRoberts reviewed (commented):
the values and ranges can't be arbitrary which seems to be the case.

garmstron42 committed:
deleted constraints.md

Deleted constraints 2-4 because I felt it would be easier to explain in 1 constraint then in several. Attempted to make changes to show each value had substance to them

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Update Device_Power.md

JesseTNRoberts reviewed (commented):
There is no reason whatsoever for you to switch 120 with the Estop. It won't be allowed in either of the devcom projects because the system doesn't warrant it.

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

Updated emergency stop to only affect the motors

JesseTNRoberts reviewed (commented):
What sort of power switch puts out 12 volts? This is not a real device that I'm familiar with.

Your BOM is not appropriately detailed.

garmstron42 committed:
Update Device_Power.md

fixed BOM

garmstron42 committed:
fixed_BOM.md

fixed the BOM

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

Fixed Drawing

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

JesseTNRoberts reviewed (commented):
This schematic looks good! 

My question now is, how do you know the amperages and voltages are sufficient?

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Good_faith_Device_Power.md

Changed a lot, had a very good meeting with the ME team and got to discuss in detail what the motor needed to accomplish and how I needed to power it. Cut the fulfilling constraints section because I looked at some older ones and most people just do in in analysis.

garmstron42 committed:
Device_Power.md

Fixed some spelling stuff, added a part i forgot about

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
final_Device_Power.md

Fixed more stuff involving the motors

JesseTNRoberts reviewed (commented):
In the design, why are there numbers and * ? 

The Estop does not stop all motion.

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

added relay system

garmstron42 committed:
Device_Power.md

fixed references stuff

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

Fixed drawings

JesseTNRoberts commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,151 @@
+
+One of the requirements in the rulebook, given to us by the customer, is that the interceptor needs to have an emergency stop that de-energizes the interceptor. This emergency stop will cut power from the AC-DC convertor for the mechanical system to the motors themselves which will de-energize the interceptor. This system will only be used as a last case option if the interceptor threatens people ...
+
+<sup>2</sup> The system shall convert 100-120 wall outlet AC voltage to 20 watts, which is required by the main/processor unit, 4.5 watts, which is required by the extra subsytem, 50 watts, which is required by the emergency stop system and a maximum of 480 Watts DC, which is required by the mechanical unit [Conceptual Design]
```
Watts are not specific. You need to give the voltage and current. Further, this needs to consider the ripple voltage that the power supplies allow through.

JesseTNRoberts reviewed (commented):
The relay diagram does not make sense. It does not match the documentation for the device. This makes me question the entire electrical schematic...

garmstron42 committed:
Update Device_Power.md

garmstron42 committed:
Add files via upload

garmstron42 committed:
Device_Power.md

Changed a lot

garmstron42 committed:
Update Device_Power.md

micahr234 commented on Documentation/Signoffs/Device_Power.md:
```diff
@@ -0,0 +1,162 @@
+To acomplish this power draw, the system will utilize the 5 volt 10 amp power supply that will also be powering the extra system and relay coil[1]. 
+
+### Emergency Stop
+
```
The estop you chose has a normally closed contact... meaning it is closed until you press the button. What does this do to you estop logic?

garmstron42 replied:
I may be thinking about this wrong, but if it is NC, then the power should be flowing into the relay, turning it on (which is good because it allows the power to go to the ME system). Then when it is pressed it will turn off the ME system. So I believe this should be correct?

garmstron42 replied:
No your right, after looking at the way I have it set up, the Estop needs to be a NO switch or I need to move the relay output to be NO. I will make those changes.

garmstron42 committed:
Add files via upload

garmstron42 committed:
Update Device_Power.md

garmstron42 commented:
I changed the relay connection to be NO because I felt this would be the safer of the two options. That way, if the e stop is messed up, the relay will always stay off.
